src/drivers/distance_sensor/vl53lxx/vl53lxx.cpp

```diff
@@ -891,16 +797,15 @@ VL53LXX::sensorTuning()
 	writeRegister(0xFF, 0x00);
 	writeRegister(0x80, 0x00);
 
-	return OK;
+	return PX4_OK;
```

Comment:
PX4_OK is not the correct return value when returning bool

Reply:
Return type modified to `int`.  Let me know if you'd prefer to see anything different.
Thanks so much for your review!